antismash/modules/smcog_trees/trees.py

```diff
@@ -151,14 +139,38 @@ def draw_tree(input_number: int, output_dir: str, tag: str) -> str:
             clade.branch_length = max_size / 20
         else:
             clade.branch_length = abs(clade.branch_length) + max_size / 20
-    # change the colour of the query gene
-    label_colors = {tag: 'green'}
+
+    handle = StringIO("")
+    Phylo.NewickIO.write([tree], handle)
+    handle.seek(0)
+    content = handle.read()
```

Comment:
wouldn't this just be `handle.getvalue()` without having to faff around with `seek()`?